src/components/Shared/DatePicker/index.js

```diff
@@ -0,0 +1,80 @@
+import Flatpickr from 'react-flatpickr';
+import moment from 'moment';
+import React, { Component } from 'react';
+
+import 'flatpickr/dist/themes/material_blue.css';
+import 'components/Shared/DatePicker/DatePicker.css';
+/**
+  Passes an estimated openDate via the "onChange" prop.
+  Unless the "Indefinite Closure" checkbox is ticked, then the crossing will not have an estimated openDate.
+**/
+export default class DatePicker extends Component {
+  stringToDate(dateString) {
+    return dateString ? moment(dateString).toDate() : null;
+  }
+
+  dateToString(date) {
+    return moment(date).format('YYYY-MM-DD');
+  }
+
+  render() {
+    const { openDate, indefiniteClosure } = this.props;
+    const flatpickrDate = this.stringToDate(openDate)
+
+    return (
+      <div className="duration-container">
+        <table className='duration-table'>
+          <tbody>
+            <tr>
+              <td>
+                Estimated Open Date:
+              </td>
+              <td>
+                <Flatpickr
+                  className='flatpickr-extra'
+                  options={{
+                    dateFormat: 'Y-m-d',
+                    minDate: 'today',
+                    enableTime: false
+                  }}
+                  value={flatpickrDate}
+                  onChange={date => {
+                    const newOpenDate = this.dateToString(date[0])
+                    this.props.onChange({
+                      indefiniteClosure: false,
+                      openDate: newOpenDate
+                    });
+                  }}
+                />
```

Comment:
I'm not the biggest fan of the UI here:
<img width="409" alt="screen shot 2018-11-07 at 4 13 31 pm" src="https://user-images.githubusercontent.com/6384156/48164437-19cc4280-e2a8-11e8-8096-c362e46230a0.png">

I think the best thing would be to get some design help on it so we can decide on what we'd like this to look like, but for now I just want to make sure that Flatpickr isn't going to lock us in to the text input box, as it might be nice to have the date not show as if we can type it in and instead have a little 📆button or something.

I was thinking it'd be cool to use our "PrettyDate" as you named it and then have a button next to it or something along those lines.

Reply:
Fair enough. I'll merge it for now, but the UI can be improved later if we want.

Reply:
Perfect! Any concerns about flatpickr making that difficult?